lib/simctl.js

```diff
@@ -32,7 +27,14 @@ async function launch (udid:string, bundleId:string):void {
 
 async function createDevice (name:string, deviceTypeId:string,
     runtimeId:string):void {
-  await simExec("create", 0, [name, deviceTypeId, runtimeId]);
+
+  let out;
+  try {
+    out = await simExec("create", 0, [name, deviceTypeId, runtimeId]);
+  } catch (e) {
+    throw new Error(`Could not create simulator. Reason: ${e.stderr.trim()}`);
```

Comment:
we should use `log.errorAndThrow` whenever we originate an error (i.e., where we create / throw the error) so that we see the log in the flow context.